packages/bits/demo/src/components/demo/paginator/paginator-test/paginator-test.component.html

```diff
@@ -0,0 +1,10 @@
+
+<div class="container">
+    <nui-paginator-basic-example></nui-paginator-basic-example>
+</div>
+
+<div>
+    <nui-paginator-adjacent-example>
+        <nui-paginator [adjacent]="2"></nui-paginator>
```

Comment:
Is this extra paginator needed?

Reply:
I added it because I would like to have _[adjacent]="2"_.

Reply:
It looks like  `nui-paginator-adjacent-example` already sets its adjacent input to 2.

Reply:
Blake, you are completely right, I forgot about it.